### PR TITLE
New from_geojson method 

### PR DESCRIPTION
**Pull request by saeranv**
@chriswmackey , (tagging @mostaphaRoudsari to keep you in loop)

This PR has my implementation of the from_geojson method, along with associated helper functions in model.py and projection.py for degree to model unit conversions. 

I've kept this as a draft PR for now because I need some clarification on how to handle the floor to floor heights for the new model Story objects. There's no information in the geojson file that specifies this, so for now I've added an argument from the user that specifies the floor to floor height, with a default of 3.5 meters for all stories. 

Let me know if there's a different way to handle this.

### CODE REVIEW

File: dragonfly/model.py [hunk 1]
```diff
@@ -96,6 +100,112 @@ def __init__(self, identifier, buildings=None, context_shades=None,
...
+            location: An optional ladybug location object with longitude and
+                latitude data defining the origin of the geojson file. If nothing
+                is passed, the origin is autocalculated as the bottom-left corner
+                of the bounding box of all building footprints in the geojson file
+                (Default: None).
+            point: A ladybug_geometry Point2D for where the location object exists
+                within the space of a scene. The coordinates of this point are
+                expected to be in the expected units of this Model (Default: (0, 0)).
+
+            # TODO: Clarification required re: correct way to handle floor heights
```
Comment (saeranv) [inline dragonfly/model.py:121]: @chriswmackey, thoughts on how to do floor heights?

**Comment by saeranv** [comment]
FYI: I just added a couple more tests to check unit conversion and floor to floor height of stories.

**Comment by chriswmackey** [comment]
Thanks, @saeranv . I will review this as soon as I get the chance. I was originally thinking that the floor-to-floor height would be computed from the building height and the number of stories but I will take a closer look at what you have and see if it makes sense.

**Comment by saeranv** [comment]
@chriswmackey you're right! I see that there's a '"maximum_roof_height"' in the geojson that I missed. I was just multiplying the floor heights by number of stories, but using the roof height makes a lot more sense.

**Comment by saeranv** [comment]
@chriswmackey I've fixed the floor_to_floor_height derivation, and taken this out of draft status, think it's ready for review.

**Comment by saeranv** [comment]
Thanks for the review @chriswmackey. Sorry for my delayed response (been working on the plenum issue), but will be sending over the PR for this tonight.

File: dragonfly/model.py [hunk 2]
```diff
@@ -96,6 +100,110 @@ def __init__(self, identifier, buildings=None, context_shades=None,
...
+                Default: 0.
+        """
+
+        with open(geojson_file_path, 'r') as fp:
+            data = json.load(fp)
+
+        # Get the project and list of building data
+        proj_data = data['project']
+        bldgs_data = [bldg_data for bldg_data in data['features']
+                      if bldg_data['properties']['type'] == 'Building']
```
[the remark this answers is not in the document]
Comment (saeranv) [reply dragonfly/model.py:146]: Makes sense, added the argument and check.

**Comment by saeranv** [comment]
@chriswmackey this update contains all the checks for the minimal geojson file. I also added an extra test just for the minimal version. Looks like everything works.

**Pull request merged**